tools/stonith_admin.c

```diff
@@ -57,7 +57,7 @@ static struct crm_option long_options[] = {
     {   "broadcast", no_argument, NULL, 'b',
         "Broadcast wherever appropriate."
     },
-    PCMK__OUTPUT_OPTIONS("text, xml"),
+    PCMK__OUTPUT_OPTIONS("html, text, xml"),
```

Comment:
I'm thinking it might be worthwhile to use something like "text (default), html, xml"